best-practices.md

```diff
@@ -657,6 +657,13 @@ So if you are writing a STAC client it is recommended to start with just support
 turn, if your data is published online publicly or for use on an intranet then following these recommendations will ensure
 that a wider range of clients will work with it.
 
+#### Unique Parent Link
+
+STAC catalogs do not support multiple parents. Links with `rel` of type `parent` and `collection` must be unique.
```

Comment:
> STAC catalogs do not support multiple parents.

This sounds like a requirement but is in best practices. Should this be moved to the spec?

Reply:
Also, there can be multiple parent links for different encodings (e.g. application/json and text/html). We should clarify that this only applies for STAC-specific media types.

Reply:
moved to spec